client/src/pages/AccountActivationPage.jsx

```diff
@@ -0,0 +1,41 @@
+import { useContext, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { AuthContext } from '../components/AuthContext.jsx';
+import { Loader } from '../components/loader/Loader.jsx';
+
+export const AccountActivationPage = () => {
+  const [error, setError] = useState('');
+  const [done, setDone] = useState(false);
+
+  const { activate } = useContext(AuthContext);
+  const { activationToken } = useParams();
+
+  useEffect(() => {
+    activate(activationToken)
+      .catch((error) => {
+        setError(error.response?.data?.message || `Wrong activation link`);
+      })
+      .finally(() => {
+        setDone(true);
+      });
+  }, []);
```

Comment:
The dependency array for useEffect is empty, which means the effect will only run once when the component mounts. If the activationToken can change, it should be included in the dependency array to ensure the effect runs whenever it changes.